internal/frontend/cuefrontend/module.go

```diff
@@ -85,6 +86,43 @@ func moduleFrom(ctx context.Context, dir, workspaceFile string, data []byte) (pk
 	}, nil
 }
 
+func (moduleLoader) NewModule(ctx context.Context, dir, workspaceFile string, w *schema.Workspace) (pkggraph.WorkspaceData, error) {
+	val, err := decodeWorkspace(w)
+	if err != nil {
+		return nil, err
+	}
+	return workspaceData{
+		absPath:        dir,
+		definitionFile: workspaceFile,
```

Comment:
New modules should always use the new filename. Let's pin `cuefrontend.WorkspaceFile` here and remove the parameter.